src/shapes/text.class.ts

```diff
@@ -608,7 +592,7 @@ export class Text<
     charStyle: any,
     forMeasuring?: boolean
   ) {
-    ctx.textBaseline = 'alphabetical';
+    ctx.textBaseline = 'alphabetic';
```

Comment:
Thanks to TS.
I am pretty sure this fixes an open bug

Reply:
i have seen this too in the past.
I think the baseline work correctly because `alphabetic` is the default and wrong values are ignored.
At least i never noticed anything wrong with baseline.